java-client-example/src/main/java/NebulaStreamTutorial.java

```diff
@@ -30,7 +30,7 @@ public static void main(String[] args) throws RESTException, IOException {
         Sink sink = query.sink(new FileSink("/tutorial/java-query-results.csv", "CSV_FORMAT", true));
 
         // The line below is necessary because of a bug when submitting queries with aggregations over Protobuf.
-        // https://github.com/nebulastream/nebulastream/issues/3429
+        // TODO https://github.com/nebulastream/nebulastream/issues/3429
```

Comment:
Should we keep this TODO? The issue is already there and we could remove the comment from the code.